src/wallet/wallet.ts

```diff
@@ -143,16 +186,45 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
    * @memberof HathorWalletServiceWallet
    * @inner
    */
-  async start() {
+  async start({ pinCode }) {
```

Comment:
Missing new parameter on docstring

Reply:
Fixed in 35809f7e9114b604a02e97799a11921286c42ee2